### ISSUE TITLE
Hide new sign up notifications

### ISSUE THREAD

**Issue by MaybeThisIsRu** [metabolist/metatext#191]
**Is your feature request related to a problem? Please describe.**
As an instance moderator, I receive around ~100-500 notifications about new sign ups. This drowns out any favourites or boosts or follow request notifications. Switching to the mentions tab doesn't help because it too shows all these notifications.

**Describe the solution you'd like**
Either a toggle to hide new sign up notifications, or just hide them permanently.

**Additional context**
![image](https://user-images.githubusercontent.com/3974019/202604942-31c33a41-26cf-416d-b9ff-8e466df9defc.jpeg)

This goes on and on for pages and pages rendering the notifications tab useless for me. :(

**Comment by abeldotam** [comment]
Yes please, +1 for this issue. This behavior is pretty annoying (even if the rest of the app is pretty awesome!)
